input/fsh/Ex_Messaging.fsh

```diff
@@ -65,7 +69,7 @@ Description: "Header for Update: Example"
 // * eventUri = "http://nchs.cdc.gov/vrdr_submission_update"
 * destination.endpoint = "http://nchs.cdc.gov/vrdr_submission"
 * source.endpoint = "https://sos.ny.gov/vitalrecords"
-* focus.reference = "Bundle/DeathCertificateDocument-Example1"
+* focus.reference = "http://www.example.org/fhir/Bundle/DeathCertificateDocument-Example1"
```

Comment:
Any reason these shouldn't all be like this for consistency?
insert addReference(focus, Bundle, DeathCertificateDocument-Example1)

Reply:
no reason, just an oversight. I was testing it out as a sanity check and forgot to change it. Thanks for catching